src/jvmMain/kotlin/io/beatmaps/api/review.kt

```diff
@@ -152,6 +152,7 @@ fun Route.reviewRoute() {
         val reviews = transaction {
             try {
                 Review
+                    .join(reviewerAlias, JoinType.INNER, Review.userId, reviewerAlias[User.id])
```

Comment:
Pretty sure this was purposefully omitted. The reviewer is always the user. You should already have that information on the profile page.